### PR TITLE
Improve URL preview formatting and image upload thumbnail size

### PR DESCRIPTION
**Pull request by williamkray**
Cherry-picked version of PR https://github.com/matrix-org/matrix-react-sdk/pull/5458 because something in github seems screwy.

### CODE REVIEW

File: res/css/views/rooms/_LinkPreviewWidget.scss [hunk 1]
```diff
@@ -19,6 +19,8 @@ limitations under the License.
     margin-right: 15px;
     margin-bottom: 15px;
     display: flex;
+    flex-direction: column;
+    max-width: 360px;
```
Comment (nadonomy) [inline res/css/views/rooms/_LinkPreviewWidget.scss:23]: We may want to increase this to 400px or so after testing.

**Review by nadonomy — APPROVED** [review]
Overall this looks great, LGTM from the design side!

**Review by jryans — APPROVED** [review]
Thanks, this seems reasonable to me code wise. 😄

**Comment by jryans** [comment]
After testing this changes for a bit on develop, discussing with @nadonomy, and reviewing community feedback [we've decided](https://github.com/vector-im/element-web/issues/16479#issuecomment-784159825) to [revert](https://github.com/matrix-org/matrix-react-sdk/pull/5677) this work for now. (See the linked decision for more details.)

**Pull request merged**